pom.xml

```diff
@@ -94,20 +94,16 @@
         <dependency>
             <groupId>org.spigotmc</groupId>
             <artifactId>spigot-api</artifactId>
-            <version>1.16.5-R0.1-SNAPSHOT</version>
+            <version>1.17.1-R0.1-SNAPSHOT</version>
             <scope>provided</scope>
         </dependency>
 
         <dependency>
-            <groupId>com.github.TheBusyBiscuit</groupId>
-            <artifactId>CS-CoreLib</artifactId>
-            <version>6e17183656</version>
+            <groupId>io.github.baked-libs</groupId>
+            <artifactId>dough-api</artifactId>
+            <version>1.0.2</version>
             <scope>provided</scope>
             <exclusions>
```

Comment:
CS-CoreLib 1 is no longer in use.
This should be removed...

Reply:
I don't follow... Didn't i remove it?

Reply:
Nope, you replaced it with dough.
Dough is not a plugin, makes no sense to have a dependency on it this way, it is shaded into Slimefun.

Reply:
Ah yes, i was wondering about why cscl was there in the first place.

Reply:
Because this wasn't updated in a long time :LUL:

Reply:
Aye. Should be fixed now